source/math/FloatBox2D.ooc

```diff
@@ -58,14 +58,14 @@ FloatBox2D: cover {
 	}
 	pad: func ~fromFloat (pad: Float) -> This { this pad(pad, pad, pad, pad) }
 	pad: func ~fromSize (pad: FloatSize2D) -> This { this pad(pad width, pad width, pad height, pad height) }
-	pad: func ~fraction (pad: Float) -> This {
-		this pad(pad * this size / 2.0f)
+	enlargeEvenly: func (fraction: Float) -> This {
```

Comment:
Well, that's the problem, I don't, so I guess it'll have to do. A comment explaining what you mean by "evenly" would be nice, though.